GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
I'm not a fan of this change. I don't think git-tfs should set user.name or user.email. If there's stuff that requires it, could git-tfs just raise an error that says "You need to set your user.name and user.email before continuing"?